BaseApi.js

```diff
@@ -173,6 +173,12 @@ const API = {
     const url = ct_url.replace('$objectPk', objectPk);
     return makePostRequest(url, data, token);
   },
+  editComment(contentTypeId, objectPk, commentPk, data, token = null) {
+    const ct_url = endpoints.comment.replace('$contentTypeId', contentTypeId);
```

Comment:
i know it is the pythonic way, but to keep consistency here i would rather do camelcase notation?
eslint should catch this, though

Reply:
I just copied this from the other api functions :) so apparently eslint didn't complain before

Reply:
i see, okay lets leave it for now